### DISCUSSION TITLE
Images for collections?

### DISCUSSION THREAD

**Discussion by eypacha** [payloadcms/payload#3283] [Q&A] [answered]
Hi everyone!

Is there a simple way to add images to collections like you do with blocks? Or would you have to develop a plugin for that? I am not referring to the images field but to the imageURL option of the blocks.

Thanks

**Comment by matthijs166** [comment, marked as answer]
The way to go is to use the relationship field ([Documentation](https://payloadcms.com/docs/fields/relationship))

My recommendation is to look and try the public demo. There is an example there on how to reference the media collection:
(https://github.com/payloadcms/public-demo/blob/master/src/blocks/Media.ts)

**Reply by eypacha** [reply]
I don't think I'm expressing myself correctly (I don't speak English natively)

In the blocks there is a property called imageURL that allows you to choose an image to be displayed in the block list
![blocks](https://github.com/payloadcms/payload/assets/5504153/79c3cde7-4e32-4914-9300-7a75a0e0afe9)

However, I cannot do the same in the collections
![collections](https://github.com/payloadcms/payload/assets/5504153/819bd2b7-51d1-4add-9d53-db9b4f99d026)

Is there a way to do that?

**Reply by matthijs166** [reply]
You could build your own dashboard components. There is at the moment no way to add icons in those dashboard tiles.

Also, payload 2.0 coming soon (I'm very excited) Don't know what will happen to the dashboard then.

This will maybe help you for building out your dashboard:
https://www.youtube.com/watch?v=ZXoi9sYk96k&t
https://www.youtube.com/watch?v=ZyfZ3HIwY94

You can use the current dashboard as starting point, and customize it until you got it how you want:
https://github.com/payloadcms/payload/blob/37fe2df2de41b4a64cbda228e6adad349aefa4c0/src/admin/components/views/Dashboard/Default.tsx#L18

Your config would like something like this:
```js
export default buildConfig({
  admin: {
    components: {
      views: {
        Dashboard: MyCustomDashboard,
      },
    },
  },
});
```

Also helpful:
https://payloadcms.com/docs/admin/components
